arc4random.c

```diff
@@ -382,9 +382,6 @@ arc4_stir(void)
 	if (0 != arc4_seed())
 		return -1;
 
-	if (!arc4_seeded_ok)
-		return -1;
```

Comment:
And we do not need `arc4_seeded_ok` anymore, correct?

Reply:
yup, looks like `arc4_seeded_ok` is not read anywhere else

Reply:
@sungjungk can you rebase the patch?

Reply:
Could you check the following rebased patch?

Reply:
I'm still seeing `arc4_seeded_ok` variable

Reply:
I removed ```arc4_seeded_ok``` variables and rebased the patch.

Reply:
Applied, thanks!